app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs

```diff
@@ -15,4 +15,9 @@ public static class Keys
         public static readonly string REDIS_KEY_PREFIX = "dyna_";
         public static readonly string DATA_PROVIDER_KEY = "ProviderList";
     }
+
+    public static class Constants
+    {
+        public static readonly string CouldNotLocate = "Could Not Locate";
+    }
```

Comment:
Can we use  `const` instead of readonly CouldNotLocate

Reply:
sure